ambari-common/src/main/python/ambari_commons/os_linux.py

```diff
@@ -84,3 +84,11 @@ def os_set_open_files_limit(maxOpenFiles):
 
 def os_getpass(prompt):
   return getpass.unix_getpass(prompt)
+
+def os_is_service_exist(serviceName):
+  systemManager = os.popen('ps --no-headers -o comm 1').read().strip()
```

Comment:
we can use this way to detect systemd: https://www.freedesktop.org/software/systemd/man/sd_booted.html

i.e. check existence of /run/systemd/system/

Reply:
Ok